ReplicationAcceptance/ReplicationAcceptance/ReplicationAcceptance.m

```diff
@@ -1581,7 +1708,8 @@ -(void) testURLConnectionChangeTrackerWithRealRemoteUsingAuthorizer
     });
     
     while (!changeTrackerStopped) {
-        [NSThread sleepForTimeInterval:5.0f];
+        [[NSRunLoop currentRunLoop] runMode:NSDefaultRunLoopMode
+                                 beforeDate:[NSDate distantFuture]];
```

Comment:
I don't know, but I think it should sleep for `5.0f` rather than `distantFuture` though will update